.nycrc.json

```diff
@@ -2,6 +2,6 @@
   "extends": "@istanbuljs/nyc-config-typescript",
   "all": true,
   "silent": true,
-  "include": "site",
-  "exclude": ["site/.next", "site/public/blocks", "site/tests"]
+  "include": "apps",
+  "exclude": ["apps/site/.next", "apps/site/public/blocks", "apps/site/tests"]
```

Comment:
@CiaranMn codecov is still empty, but I've run out of ideas. Not sure what else to try. I guess we'll need to fix codecov in another PR, unless it's some kind of a bug that fixes itself. We moved files in this commit, which may somehow affect path resolution.


When generating `coverage/lcov.info` locally, I get expected coverage, e.g.:

```
SF:apps/site/sentry.client.config.ts
FNF:0
FNH:0
DA:6,9
DA:8,9
DA:14,9
LF:3
LH:3
BRDA:11,0,0,9
BRDA:11,0,1,9
BRF:2
BRH:2
```

`DA:6,9` means ‘line 6 covered 9 times’.

<img width="627" alt="Screenshot 2022-10-07 at 00 55 42" src="https://user-images.githubusercontent.com/608862/194438840-d213db17-ac76-4fd5-8ff2-18f1e8d7df69.png">

When I upload this file as an artifact via codecov action, `DA:6,9` becomes `DA:6,0`:

```
SF:apps/site/sentry.client.config.ts
FNF:0
FNH:0
DA:6,0
DA:8,0
DA:14,0
LF:3
LH:0
BRDA:11,0,0,0
BRDA:11,0,1,0
BRF:2
BRH:0
end_of_record
TN:
```

🤷

Reply:
Test coverage was reported on main at ≈40%, which confirms a potential bug in codecov. I’m restoring the original ≈60% in #654